package.json

```
@@ -171,6 +173,7 @@
     "fs-extra": "9.1.0",
     "gaze": "1.1.3",
     "glob": "7.1.6",
+    "http-server": "^13.0.1",
```

nit: we should always use exact versions:
```suggestion
    "http-server": "13.0.1",
```